Resources/doc/fosuser-bundle.md

```diff
@@ -1,3 +1,94 @@
 # FOSUser Bundle integration
 
 This bundle is shipped with a bridge for the [FOSUserBundle](https://github.com/FriendsOfSymfony/FOSUserBundle). If the FOSUserBundle is enabled, this bridges registers to the persist, update and delete events to pass user objects to the UserManager, before redispatching the event. 
+
+## Creating a `User` entity with serialization groups
+
+First register the following service:
+
+```yaml
+# app/config/services.yml
+
+resource.user:
+        parent:    "api.resource"
+        arguments: [ "AppBundle\\Entity\\User" ]
+        calls:
+            -      method:    "initNormalizationContext"
+                   arguments: [ { groups: [ "user_read" ] } ]
+            -      method:    "initDenormalizationContext"
+                   arguments: [ { groups: [ "user_write" ] } ]
+        tags:      [ { name: "api.resource" } ]
+```
+
+Then create your User entity with serialization groups:
+
+```php
+<?php
+
+namespace AppBundle\Entity;
+
+use FOS\UserBundle\Entity\User as BaseUser;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ *
+ * @UniqueEntity("email")
+ * @UniqueEntity("username")
```

Comment:
Are the `@UniqueEntity` necessary? It's not bothersome since it's and example, but maybe it would be better to keep with FOSUserBundle example.